examples/SelfEncryptorExample.rs

```diff
@@ -0,0 +1,55 @@
+extern crate self_encryption;
```

Comment:
How does a person use this example ?
Is the naming convention correct for the example? should it not be self_encryptor_example ?